.github/workflows/python-package.yml

```diff
@@ -43,4 +43,34 @@ jobs:
     - name: Test with pytest
       run: |
         pip install .
-        pytest
\ No newline at end of file
+        pytest
+
+  publish-package:
+    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
+
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+        python-version: [ "3.9" ]
```

Comment:
I think we should test this on 3.10 and 3.11.
[This workflow](https://github.com/prime-slam/octreelib/actions/runs/6761073417/job/18375550671) failed on 3.11 but completed on 3.9 and 3.10
The issue was a mutable field in a dataclass which are not allowen in 3.11:

https://github.com/prime-slam/octreelib/pull/14/commits/155d2a6865b98a8258a6903d9b64fea08896df91

If we only want to test one version, I suggest it to be 3.11
However, I think it is safer to test all 3.9, 3.10, 3.11